packages/python-packages/apiview-gpt/ci.yml

```diff
@@ -14,22 +11,17 @@ pr:
   branches:
     include:
       - main
-      - feature/*
-      - release/*
-      - hotfix/*
   paths:
     include:
       - packages/python-packages/apiview-gpt
 
-extends:
-  template: /eng/pipelines/templates/stages/archetype-sdk-tool-python.yml
-  parameters:
-    PythonVersion: '3.10'
-    PackagePath: 'packages/python-packages/apiview-gpt'
-    FeedName: 'public/azure-sdk-for-python'
-    ArtifactName: 'apiviewcopilot'
-    PackageName: 'Python API View Copilot'
-    TestSteps:
-    - script: |
-        echo "Pass Check Enforcer!"
-      displayName: 'Pass Check Enforcer'
\ No newline at end of file
+steps:
+  - script: |
+      zip -r app.zip .
+    displayName: 'Package app into a zip file'
```

Comment:
this step should include ` workingDirectory:` to compress only apiviewgpt project directory. Currently it will zip everything in azure-sdk-tools repo. reference:  [workingDirectory](https://github.com/Azure/azure-sdk-tools/blob/main/src/dotnet/APIView/apiview.yml#L88)

Also set pathtopublish in below task accordingly so it gets published.